app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -556,6 +556,35 @@ This pull request is not suitable for automatic merging in its current state.
       ]);
     });
 
+    test('Does not merge if non member does not have at lease 2 member reviews', () async {
```

Comment:
trivial nit: lease -> least

Reply:
Please add a test that explicitly tess the author self approving their PR.

Reply:
changed lease -> least and added a test for self approving.